packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -310,7 +342,7 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
 
   // load scene content
   useLayoutEffect(() => {
-    if (sceneContent?.length > 0) {
+    if (sceneContent) {
```

Comment:
it was initialized with empty string. but I've changed the initial value to be undefined now. it can also be type of `ISceneDocumentInternal` that won't have length